content/docs/user-guide/project-config/cli-reference.md

```diff
@@ -6,7 +6,7 @@ weight: 200
 toc: true
 ---
 
-The `o3de` Python script lets you configure your project using the command line interface (CLI). The script aggregates all of the functions from lower-level scripts in Open 3D Engine (O3DE), so you can access them from a single point. Find the script in your engine directory at `/scripts/o3de.py`.
+The `o3de` Python script enables you to configure your O3DE environment and its objects, including engines, projects, and Gems, from a command line interface (CLI). The script aggregates all of the functions from lower-level scripts in Open 3D Engine (O3DE), so you can access them from a single point. Find the script in your engine directory at `/scripts/o3de.py`.
```

Comment:
```suggestion
Use the `o3de` Python script to configure your O3DE environment and its objects, including engines, projects, and Gems, from a command line interface (CLI). The script aggregates all of the functions from lower-level scripts in Open 3D Engine (O3DE), so you can access them from a single point. Find the script in your engine directory at `/scripts/o3de.py`.
```
Upon further review for this intro, I recommend this slight wording change. The style guide advises against using either [enable](https://alpha-docs-aws.amazon.com/awsstyleguide/latest/styleguide/dictionary.html#style-enable) or [let](https://alpha-docs-aws.amazon.com/awsstyleguide/latest/styleguide/dictionary.html#style-let) "to refer to making something in an AWS service or feature possible for the customer."